### PR TITLE
Fixing issue #507

### PR DESCRIPTION
**Pull request by geduardo**
Switching the matrices and trying to fix the spacing

### CODE REVIEW

**Comment by geduardo** [comment]
I don't know if the equation for the full construction should be also reformatted. Having to scroll to see the full expression seems a bit awkward to me when it could be split into different lines.

**Comment by cgranade** [comment]
@geduardo: Thanks for fixing this! I agree, having to scroll isn't great; I suspect that there's a similar `\\\` versus `\\\\` issue on that line as well.

**Comment by geduardo** [comment]
Exactly, all of them were `///`. I've checked in the rest of the pages of this section to look for more versions of this error and didn't find anything. So I think it's now ready to merge.

File: articles/libraries/chemistry/concepts/jordan-wigner.md [hunk 1]
```diff
@@ -45,10 +45,10 @@ Thus, by interspersing $Z$ operators into the construction of the operator, we c
...
 \begin{align}
-a^\dagger_1 &= \left(\frac{X-iY}{2}\right)\otimes 1 \otimes 1 \otimes 1 \otimes \cdots \otimes 1,\\\
-a^\dagger_2 &= Z\otimes\left(\frac{X-iY}{2}\right)\otimes 1\otimes 1 \otimes \cdots \otimes 1,\\\
-a^\dagger_3 &= Z\otimes Z\otimes \left(\frac{X-iY}{2}\right)\otimes 1 \otimes \cdots \otimes 1,\\\
-&\vdots\\\
+a^\dagger_1 &= \left(\frac{X-iY}{2}\right)\otimes 1 \otimes 1 \otimes 1 \otimes \cdots \otimes 1,\\\\
+a^\dagger_2 &= Z\otimes\left(\frac{X-iY}{2}\right)\otimes 1\otimes 1 \otimes \cdots \otimes 1,\\\\
+a^\dagger_3 &= Z\otimes Z\otimes \left(\frac{X-iY}{2}\right)\otimes 1 \otimes \cdots \otimes 1,\\\\
+&\vdots\\\\
 a^\dagger_N &= Z\otimes Z\otimes Z\otimes Z \otimes \cdots \otimes Z\otimes \left(\frac{X-iY}{2}\right). \tag{★}\label{eq:JW}
```
Comment (cgranade) [inline articles/libraries/chemistry/concepts/jordan-wigner.md]: It looks like `\tag{★}` is breaking the spacing here, plus we never use the `eq:JW` reference anywhere.
```suggestion
a^\dagger_N &= Z\otimes Z\otimes Z\otimes Z \otimes \cdots \otimes Z\otimes \left(\frac{X-iY}{2}\right). \label{eq:JW}
```

**Pull request merged**